borrowing/models.py

```diff
@@ -1,5 +1,42 @@
 from django.db import models
+from rest_framework.exceptions import ValidationError
+
+from library.models import Book
+from library_service import settings
 
 
 class Borrowing(models.Model):
-    pass
+    borrow_date = models.DateField(auto_now_add=True)
+    expected_return_date = models.DateField()
+    actual_return_date = models.DateField(null=True, blank=True)
+    book = models.ForeignKey(
+        Book,
+        on_delete=models.CASCADE,
+        related_name="borrowings",
+    )
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
```

Comment:
maybe better to use get_user_model()
initalize User = get_user_model()
and use User here

Reply:
Using settings.AUTH_USER_MODEL in the ForeignKey field is generally preferred over using get_user_model()